src/components/icon/assets/integrationObservability.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import * as React from 'react';
+const OuiIconIntegrationObservability = ({ title, titleId, ...props }) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 12 12"
```

Comment:
@canascar Note that the integration* icons appear to have been created on a 12x12 canvas. For this reason, I left off any explicit width/height properties. It should work fine, but in the future, for consistency, all icons should be built  to be 16x16 by default.